api-client/lib/api/client.rb

```diff
@@ -5,11 +5,16 @@
 require "active_support/cache"
 require "active_support/notifications"
 require "get_into_teaching_api_client"
+require "faraday_middleware/circuit_breaker"
 
 require "api/client/version"
 require "api/extensions/get_into_teaching_api_client/configuration"
 require "api/extensions/get_into_teaching_api_client/api_client"
 
+module GetIntoTeachingApiClient
```

Comment:
Just my preference but could put this on a single line to match the other statements:

```
class GetIntoTeachingApiClient::CircuitBrokenError < RuntimeError; end
```